srcpkgs/fail2ban/template

```diff
@@ -10,12 +10,13 @@ maintainer="necrophcodr <necrophcodr@necrophcodr.me>"
 license="GPL-2.0-only"
 homepage="http://www.fail2ban.org/"
 distfiles="https://github.com/${pkgname}/${pkgname}/archive/${version}.tar.gz"
-checksum=71d2a52b66bb0f87ac3812246bdd3819ec561913cd44afd39130a342f043aa6d
+checksum=383108e5f8644cefb288537950923b7520f642e7e114efb843f6e7ea9268b1e0
 conf_files="
  /etc/fail2ban/fail2ban.conf
  /etc/fail2ban/jail.conf
  /etc/fail2ban/action.d/*.conf
  /etc/fail2ban/filter.d/*.conf"
+make_dirs="/var/lib/fail2ban 0700 root root"
```

Comment:
Explain in your commit message why this is now necessary.

Reply:
I don't know if it is. In current version this directory is not created so the first run will fail, because it can't create default database `/var/lib/fail2ban/fail2ban.sqlite3`. I've added this as a convenience, but maybe previous maintainer of this package did it on purpose? Please advise should I add it or not.

Reply:
Do leave it in, then. Just mention that this makes first startup work out of the box.

Reply:
Did it in https://github.com/void-linux/void-packages/pull/32303, this one I had to close because of my stupidity.